.circleci/config.yml

```diff
@@ -139,3 +157,23 @@ workflows:
           requires:
             - approve
             - release
+      - build_cdn:
+          filters:
+            <<: *release_filter
```

Comment:
Do we want to filter by branch? I was under the impression launchbay will take care of it from now onwards...

Reply:
Hrm I supposed we don't need the filter here because this task requires `release` task to complete, which already has a branch filter. But the idea is that it `release` and `build_cdn` can be skipped on PR build.

Reply:
Oh! because you want to keep the release numbers. So you only want to release certain versions. This is different from the rest of our apps. I guess there's no problem in doing it this way 🤷

Reply:
Cool. Anyway, I've decided to cleanup the filter regardless. 🙂